.gitignore

```diff
@@ -13,6 +13,8 @@ project/boot/
 project/plugins/project/
 .bloop
 
+_site/
```

Comment:
out of curiosity, where does this come from? I mean, how was `_site` generated? I've never seen it